### PR TITLE
add translations

### PR DESCRIPTION
**Pull request by stardustichi**
a more complete version of #62 , this is super late might as well wait for tomorrow patch lol

### CODE REVIEW

File: src/data/uma-title.csv [hunk 1]
```diff
@@ -17,10 +17,10 @@
 "[Maverick]","[Maverick]"
 "[ロード・オブ・エンペラー]","[Road of Emperor]"
 "[エンプレスロード]","[Empress Road]"
-"[クエルクス・キウィーリス]",""
+"[クエルクス・キウィーリス]","[Quercus Kiwiris]"
```
Comment (FabulousCupcake) [inline src/data/uma-title.csv:20]: We talked about this and there's a better version of this but I can't remember

Comment (stardustichi) [reply src/data/uma-title.csv:20]: i dont even know what this one translate to, but let me know

Comment (FabulousCupcake) [reply src/data/uma-title.csv:20]:
```suggestion
"[クエルクス・キウィーリス]","[Quercus Civilis]"
```

Comment (FabulousCupcake) [reply src/data/uma-title.csv:20]: > Quercus Civilis A Crown of Oak-branches, given to him that saved a Citizen

Credits to a certain discord user in uma discord

**Comment by FabulousCupcake** [comment]
@stardustichi could you resolve the merge conflicts?

**Review by FabulousCupcake — APPROVED** [review]
lgtm

**Comment by FabulousCupcake** [comment]
Thanks!

**Pull request merged**